pyproject.toml

```
@@ -1,10 +1,114 @@
-[tool.black]
-line_length = 120
+[build-system]
+requires = ["setuptools", "wheel"]
+
+[project]
+name = "hallux"
+version = "0.0.1"
+authors = [
+  { name="Hallux Team", email="dmitry@hallux.dev" },
+]
+description = "Fixing project issues automagically with AI."
+readme = "README.md"
+license = {file = "LICENSE.txt"}
+requires-python = ">=3.8, <4.0"
+dependencies = [
+  'ruff==0.0.272',
+  'mypy==1.4.1',
+  'black==23.3.0',
+  'openai==0.27.8',
+  'pyyaml==6.0',
+  'types-PyYAML==6.0.12.11',
+  'PyGithub==1.59',
+  'isort==5.12.0',
+  'wheel==0.40.0',
+  'pyinstaller==5.13.0',
+]
+
+classifiers = [
+  "Development Status :: 4 - Beta",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
+  "Topic :: Scientific/Engineering :: Artificial Intelligence",
+]
+
+[project.optional-dependencies]
+test = [
+    "pytest >= 7.3.1",
+    'pytest-cov==4.1.0',
+]
+
+
+[project.scripts]
+hallux = 'src.hallux:app'
```

не уверен, но наверно должно быть 
```suggestion
hallux = 'src.hallux:main'
```

---

fixed, worked!